README.md

```diff
@@ -64,3 +66,6 @@ According to SemVer, anything in the public API can change at any time before ve
 
 For more information on Tendermint Core and pointers to documentation for Tendermint visit
 this [repository](https://github.com/tendermint/tendermint).
+
+### Careers
+We are hiring Go engineers! Join us in building the future of blockchain scaling and interoperability. [Apply here](https://angel.co/company/celestialabs/jobs).
```

Comment:
I'd either suggest moving the Careers section above the Resources one, or, alternatively make it not a sub-section of Resources but one of the parent section. The latter could be achieved by:
```suggestion
## Careers
We are hiring Go engineers! Join us in building the future of blockchain scaling and interoperability. [Apply here](https://angel.co/company/celestialabs/jobs).
```